### ISSUE TITLE
Can't get VScode work correctly

### ISSUE THREAD

**Issue by thibaudbrg** [htpcBeginner/docker-traefik#255]
Hello, I'm struggling with adding the VScode docker-compose. I'm following the instructions [here](https://www.smarthomebeginner.com/wordpress-on-docker-traefik/), but there are some sentences that I think I didn't understand correctly. 

The code I use is basically the same as the one in the repo:

  ```
vscode:
    <<: *common-keys-core # See EXTENSION FIELDS at the top
    image: codercom/code-server:latest
    container_name: vscode
    volumes:
      - $USERDIR/server:/home/coder/server
      - $DOCKERDIR:/home/coder/docker
      - $DOCKERDIR/appdata/vscode:/home/coder
    environment:
      PASSWORD: $VSCODE_PASSWORD
      # Run as root first, create the directories, then change permissions to user:docker and 775. Disable run as root below.
      user: $PUID:$PGID
      #user: "0"
      DOCKER_HOST: tcp://socket-proxy:2375
    labels:
      - "traefik.enable=true"
      ## HTTP Routers
      - "traefik.http.routers.vscode-rtr.entrypoints=https"
      - "traefik.http.routers.vscode-rtr.rule=Host(`code.$DOMAINNAME1`)"
      ## Middlewares
      - "traefik.http.routers.vscode-rtr.middlewares=chain-oauth@file"
      ## HTTP Services
      - "traefik.http.routers.vscode-rtr.service=vscode-svc"
      - "traefik.http.services.vscode-svc.loadbalancer.server.port=8080"
```

So when I'm following these instructions:
![Screenshot 2022-03-06 182711](https://user-images.githubusercontent.com/73235385/156934507-a8bc52f3-7b7d-478f-ae18-f52501629a2a.png)

I suppose during the "tricky" part, I have to do smth like: `sudo chmod -R 775 /volume1/docker/appdata/vscode/` but still, even after restarting the container the logs are:
![Screenshot 2022-03-06 183119](https://user-images.githubusercontent.com/73235385/156934678-5fb7586e-f949-4348-8b04-bf601ba5838a.png)

I don't get what is this folder /home/coder/docker, do I have to manually create it (it doesn't work)? And basically where is the folder home? I only have the folder /volume1/homes/
Do I have to create a folder /home/ in the $DOCKERDIR directory?

**Comment by Maynardnaze** [comment]
I ended up using code server and it just works

**Comment by thibaudbrg** [comment]
I will try, thanks

**Comment by thibaudbrg** [comment]
Okay, I made a little bit of work but I still did'n find a solution
I try to run theses lines of code:
```
sudo setfacl -Rdm g:docker:rwx ~/docker
sudo chmod -R 775 ~/docker
```
but I don't know why, `setfacl `is not a recognized command in synology... So I tried to allow permissions from DSM, and that's what I did.

![Screenshot 2022-03-12 153225](https://user-images.githubusercontent.com/73235385/158022120-944e1128-6e44-4535-bede-f5580b393802.png)
The "Customize" instances apperead with the previous `chmod 775` I ran and even If I now change the permissions on DSM (randomly check/uncheck permission boxes) "customizes" words stay.

Also, from File Station, my user seems to have all the permissions needed:
![Screenshot 2022-03-12 153622](https://user-images.githubusercontent.com/73235385/158022291-7ebb699a-d996-4c1f-aa29-270d4dc75c91.png)
![Screenshot 2022-03-12 153826](https://user-images.githubusercontent.com/73235385/158022318-89bc6dee-dc65-4b98-bd8b-c6b40e6bd1fe.png)
 
 So I'm a bit lost... I did a lot of research to see if it was possible to find a way to use the `setfacl `command on synology, but without success. If someone has managed to configure VSCode as explained in the HtpcBeginner's tuto, I'd very appreciate some help
